tests/e2e/utils/lib.js

```diff
@@ -1,12 +1,5 @@
-export const waitForSelector = async function ( page, selector ) {
-	return await page.evaluate(
```

Comment:
This implementation just waited for the selector to run but it didn't wait on a positive result. I think this was supposed to be a call to `waitForFunction` but it should have just been using `waitForSelector` rather than a helper.